app/models/resource.rb

```diff
@@ -1,10 +1,11 @@
 class Resource < ActiveRecord::Base
   belongs_to :conference
+  validates :name, :quantity, presence: true
```

Comment:
numerical fields should probably be validated for their numericality as well - should I be able to enter -5 as quantity or used?